<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_employees</class>
 <widget class="QMainWindow" name="add_employees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление сотрудника</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../static/resource.qrc">
    <normaloff>:/img/img/Logotip.png</normaloff>:/img/img/Logotip.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>351</width>
      <height>741</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="first_name_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Введите фамилию сотрудника</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="last_name_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>Введите имя сотрудника</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="patronymic_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>Введите отчество сотрудника</string>
     </property>
    </widget>
    <widget class="QComboBox" name="role_combobox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <item>
      <property name="text">
       <string>Администратор</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Официант</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Повар</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="status_combobox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <item>
      <property name="text">
       <string>Работает</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Уволен</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Временно</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="first_name_log">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>450</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Введите логин</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>520</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid rgb(153, 102, 51);
background-color: rgb(255, 255, 255);
border-radius: 20px;</string>
     </property>
     <property name="placeholderText">
      <string>Введите пароль</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save_empl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>590</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 12pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_back_empl">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>660</y>
       <width>311</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: rgba(153, 102, 51);
    border: 2px solid rgba(100, 50, 51);
    border-radius: 20px;
    color: white;
    font-weight: bold;
    font-size: 9pt;
}
QPushButton:hover {
    background-color: rgba(73, 140, 81);
}
QPushButton:pressed {
    background-color: rgba(73, 140, 81);
}
</string>
     </property>
     <property name="text">
      <string>Назад к главной панели &quot;Сотрудник&quot;</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавление сотрудника</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Авторизация в системе</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../static/resource.qrc"/>
 </resources>
 <connections/>
</ui>
